### ISSUE TITLE
Proton Version Not Working

### ISSUE THREAD

**Issue by GeorgeCyriac** [black7375/Firefox-UI-Fix#428]
**Describe the bug**
When I performed the [latest update](https://github.com/black7375/Firefox-UI-Fix/releases/tag/v5.2.1), the **Proton** version of the interface appears as the **Lepton** version.
I use the **Proton** version, but no matter what I do, it always displays the **Lepton** UI.

**Expected behaviour**
The _CSS_ file should give me the **Proton** version of the UI instead of the default **Lepton**.

**Screenshots**
![Screenshot (131)](https://user-images.githubusercontent.com/98342014/176966424-1172fce1-0358-439f-9a41-48ece4f2c3d3.png)
I have downloaded and tried with all the versions, but none display the **Proton** version.

**Environment:**

 - Distribution
   - [x] [Original Lepton](https://github.com/black7375/Firefox-UI-Fix)
   - [x] [Lepton's photon style](https://github.com/black7375/Firefox-UI-Fix/tree/photon-style)
   - [x] [Lepton's proton style](https://github.com/black7375/Firefox-UI-Fix/tree/proton-style)
 - Firefox Version: 102.0
 - OS:
   - [ ] Linux
   - [x] Windows
   - [ ] Mac
   - [ ] Other: 

**`user.js` setup**
Using the default **user.js** provided by the download itself.

**Comment by black7375** [comment]
Try to use `proton-style`'s `user.js`
https://github.com/black7375/Firefox-UI-Fix/blob/proton-style/user.js

**Comment by thefirstmogs** [comment]
I'm also having this issue, but it only arose after I'd applied the Lepton theme then reapplied the Proton style. I first tried out the Proton style and it worked fine. Applying the Lepton style, then applying the Proton style again made the Lepton style persist. Changing the `user.js` file to the Proton style did make a difference to the css, but the tab bar turned into [this](https://i.imgur.com/lppZFDo.png).

**Comment by black7375** [comment]
Yes, before the recent commit, there was a conflict problem using the settings of other distributions.
From the next release, we'll be able to reduce mistakes.
https://github.com/black7375/Firefox-UI-Fix/commit/217af2eda3d56e9b32ef92cef07df75a2eeabf9d

**Comment by GeorgeCyriac** [comment]
I managed to resolve this bug. Since I tried all the versions of **Lepton**, the various _user.js_ files corrupted the settings in **about:config**. Deleting the preferences made by the _user.js_ and reinstalling **Lepton** fixed the issue. Thanking you for the prompt response.


> Yes, before the recent commit, there was a conflict problem using the settings of other distributions. From the next release, we'll be able to reduce mistakes. [217af2e](https://github.com/black7375/Firefox-UI-Fix/commit/217af2eda3d56e9b32ef92cef07df75a2eeabf9d)

**Comment by Drakkai** [comment]
Hi, how to fix the issue? After applying last update the tabs look like in lepton version. I've tried to reinstall the firefox but the result is the same. 
![proton_firefox](https://user-images.githubusercontent.com/2467051/179502462-dfebe58b-de4d-4728-ada8-4c50703c1f5d.png)
n

**Comment by black7375** [comment]
@Drakkai Try to use this config file!!
https://github.com/black7375/Firefox-UI-Fix/blob/proton-style/user.js

**Comment by Drakkai** [comment]
@black7375 it didn't help. I'm using git to download and update the files.

**Comment by Drakkai** [comment]
> To explain in more detail, even if you update with Git, the setting file will not be updated.
> 
> Because of the path of the setting file. https://github.com/black7375/Firefox-UI-Fix/wiki/Installation-Guide#3-copy-files
> 
> `user.js` file should be located in `<FIREFOX_PROFILE>/user.js`, not `<FIREFOX_PROFILE>/chrome/user.js`.
> 
> tldr; Copy the contents of file https://github.com/black7375/Firefox-UI-Fix/blob/proton-style/user.js in location `<FIREFOX_PROFILE>/user.js`.

Yes, I know. I copied the file you provided (it's the same as I downloaded via git before) and that didn't help to fix the problem. Now I did an update on my other computer and the result is the same, the tabs have changed from proton to lepton.

**Comment by black7375** [comment]
I recently updated the explicit setting value to resolve the configuration conflict. https://github.com/black7375/Firefox-UI-Fix/commit/6766fd1afe3816472dc2fc245d75febfcbef00cb
Copy it again, and try restarting it please.

**Comment by black7375** [comment]
There was a mistake as a setting.
Thank you for testing.

**Comment by Drakkai** [comment]
Everything looks fine, thanks a lot!
